test-suite/Makefile

```diff
@@ -34,20 +34,24 @@ include ../Makefile.common
 
 # Default value when called from a freshly compiled Coq, but can be
 # easily overridden
-LIB := ..
+
 BIN := $(shell cd ..; pwd)/bin/
 COQFLAGS?=
 
-coqc_boot := $(BIN)coqc -coqlib $(LIB) -boot -q -test-mode -R prerequisite TestSuite $(COQFLAGS)
-coqc := $(BIN)coqc -coqlib $(LIB) -R prerequisite TestSuite $(COQFLAGS)
-coqchk := $(BIN)coqchk -coqlib $(LIB) -R prerequisite TestSuite
+ifeq ($(COQLIB),)
+  # This method of setting `pwd` won't work on win32 OCaml
```

Comment:
I think when `$(LOCAL)` is `true` (from config/Makefile) it's fine to leave COQLIB unset and that would work on windows.

Reply:
Not for dune I think, my plan is to try to remove local tho. IMHO the build layout is similar enough as not to be worth it.

Reply:
>Not for dune I think

dune does `make COQLIB=...` so the `COQLIB := ...` in the makefile is ignored.

Reply:
Yeah I am never so sure about that overwriting semantics, they seem to scare me; what happens if `COQLIB` is set by an envvar?

Reply:
Setting inside the makefile overrides env vars.